tests/test_slim_engine.py

```diff
@@ -27,8 +27,9 @@ def count_mut_types(ts):
     mut_info = {}
     for mut in ts.mutations():
         for j, md in zip(mut.derived_state.split(","), mut.metadata["mutation_list"]):
```

Comment:
I think that all alleles should now be single nucleotides, so I'm surprised this works?

Reply:
yeah, AFAICT none of these tests actually generate multiple alleles per site. I modified the existing tests as little as possible to try to avoid introducing stochastic failures-- this code works with single or stacked alleles, coded as integers or nucleotides.

Reply:
In case it wasn't clear, what I mean is that we might now have `mut.derived_state` equal to `"A"` while `mut.metadata["mutation_list"]` a list of length greater than one. The `zip` won't fail but the test probably would.

Reply:
you're totally right -- it'll fail with stacked mutations after conversion to nucleotides. Setting the stacking policy like you suggest should do the trick.

Reply:
... or, enumerating `mutation_list` rather than zipping it with `derived_state` makes these tests pass regardless of whether the derived state is a stack or a single allele.